app/services/GuidanceService.scala

```diff
@@ -175,5 +179,12 @@ class GuidanceService @Inject() (
         )
     }
 
-  private def isAuthenticationUrl(url: String): Boolean = url.drop(1).equals(core.models.ocelot.Process.SecuredProcessStartUrl)
+  private def isAuthenticationUrl(url: String): Boolean = url.drop(1).equals(SecuredProcess.SecuredProcessStartUrl)
+
+  private def map[A](f: Retrieve[A])(g: A => A)(implicit ec: ExecutionContext): Retrieve[A] = {
```

Comment:
It may be appropriate, but map is a confusing name for this method.

Reply:
its a standard implimentation of map(), as you'd find on List , Option etc, so intentionally called map() to indicate this

Reply:
It is certainly an interesting function.